contracts/contracts/ValidatorAuction.sol

```diff
@@ -44,19 +45,25 @@ contract ValidatorAuction is Ownable {
     constructor(
         uint _startPriceInWei,
         uint _auctionDurationInDays,
-        uint _numberOfParticipants,
+        uint _minimalNumberOfParticipants,
+        uint _maximalNumberOfParticipants,
         DepositLocker _depositLocker
     ) public
     {
         require(_auctionDurationInDays > 0, "Duration of auction must be greater than 0");
-        require(_numberOfParticipants > 0, "Number of participants must be greater 0");
+        require(_minimalNumberOfParticipants > 0, "Minimal number of participants must be greater than 0");
+        require(_maximalNumberOfParticipants > 0, "Number of participants must be greater than 0");
+        require(_minimalNumberOfParticipants < _maximalNumberOfParticipants, "The minimal number of participants must be smaller than the maximal number of participants.");
```

Comment:
I think we should allow it do be set to the same value, to basically have the old functionality. Does that create problems?